script.js

```diff
@@ -0,0 +1,10 @@
+function openNav() {
+  document.getElementById('mySidenav').style.width = '100%';
+}
+
+function closeNav() {
```

Comment:
- [ ] OPTIONAL
Nice to prefer `arrow functions` over `regular functions` to enhance simpler code